test/mason/mason-argparse/MasonArgParseExample.good

```diff
@@ -38,3 +44,9 @@ one
 two
 subBoolFlag: true
 subcommand positional value: bValue
+++
```

Comment:
It seems surprising to me that `passedThrough.values()` includes the `++`. At the least this is worth some docs comments in `addPassThrough` but you might consider taking it out for pass throughs.

I guess in the case of a subcommand it corresponds to `argv[0]` being the name of the program? (I.e. it is now the name of the subcommand)?

Reply:
> I guess in the case of a subcommand it corresponds to argv[0] being the name of the program? (I.e. it is now the name of the subcommand)?

Yes, exactly that. I thought that would be a consistent way to approach it.


> It seems surprising to me that passedThrough.values() includes the ++.

Yeah, I was on the fence about that one too. Your comment pushed me to create a new sub-class and treat the delimiter for passthroughs differently than the subcommand.